.github/workflows/ci.yml

```diff
@@ -2,27 +2,30 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [jenkins/node-16-1b2a56c]
```

Comment:
update pull_request branches list before merging

Reply:
@aht007 specify the branch master for pull_request event.

Reply:
We don't need to do that since leaving it empty serves the same purpose as well as it will trigger the checks for any other PRs too that are pointing towards some other base branch